components/AllocatedWorkers/DeallocateWorkerDetails/DeallocatedWorkerDetails.jsx

```diff
@@ -5,29 +5,33 @@ import PropTypes from 'prop-types';
 import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
 import { Button, TextArea } from 'components/Form';
 import Spinner from 'components/Spinner/Spinner';
-import { deleteAllocatedWorkers } from 'utils/api/allocatedWorkers';
+import { deleteAllocatedWorker } from 'utils/api/allocatedWorkers';
 
 const DeallocatedWorkersDetails = ({
   personId,
+  id,
   allocatedWorker,
   firstName,
   lastName,
```

Comment:
`firstName` and `lastName` are not longer returned by the `API`, we can pass it down from the parent component. Or, even rid of the person bit 😅

Reply:
A good idea is to fix the component `propTypes` that at the moment are incomplete, and set `isRequired` to the meaningful ones.

Reply:
:thumbsup: decided on removing the person section since it doesn't really bring any value as the user would know what case their deallocating. made all the `propTypes` required beside `workerType` as I don't think that is very meaningful compared to knowing their name or what team their in